resources/config/config.yml

```diff
@@ -0,0 +1,47 @@
+imports:
+- { resource: services/*.yml }
+
+framework:
+    # We currently do not depend on this setting, yet it has to be present.
```

Comment:
Minor detail, but because of [`.editorconfig`](https://github.com/opencfp/opencfp/blob/0a968c46f36e736858fc9adb1aa2f8ee6560f7b5/.editorconfig#L13-L14) where we configure indentation for YAML files with 2 spaces, can we adjust here?

Reply:
Done.